CHANGELOG.md

```diff
@@ -11,6 +11,16 @@ Our versioning strategy is as follows:
 
 ## Unreleased
 
+### 🛠 Breaking Changes
+
+* `[sitecore-jss-rendering-host]` devServer is retired. renderingHostServer is the only way to use rendering-host with react from now on.
+
+### 🎉 New Features & Improvements
```

Comment:
Seems like changes not resolved as expected, below you can see same lines (check line 26), and 21.1.0 should be below "jss scaffold" feature